packaging/nmstate.spec

```diff
@@ -23,7 +23,7 @@ provider support on the southbound.
 
 %package -n python3-%{libname}
 Summary:        nmstate Python 3 API library
-Requires:       NetworkManager-libnm >= 1:1.22.8
+Requires:       NetworkManager-libnm >= 1:1.22.10
```

Comment:
Is this version available on downstream? If not, this may present a problem.

Reply:
Yes. Fedora 31 does.

Reply:
I was referring to RHEL.

Reply:
If we are waiting on RHEL, we will never got a release in half year.

Reply:
BTW, RHEL 8.1 does not have 1.22.8 we are currently depending on. I don't know why we are wasting time on waiting RHEL for this.

Reply:
You changed the dependency and I pointed out that with this spec you cannot create a build for downstream on any RHEL.

You can simply say that there is no need to do so for master, or that you have a workaround or that you can place a condition in the spec or that I am wrong and there is no problem.

Reply:
For RHEL downstream, they should use copr repo https://copr.fedorainfracloud.org/coprs/networkmanager/NetworkManager-1.22/ or master of NM.

The master branch of nmstate is not targeted for NM shipped in current RHEL release.